tests/test_cam_objective.py

```diff
@@ -17,17 +21,21 @@ def test_crop_camera(main_window: MainWindow):
     assert len(main_window.viewer.layers) == 1
 
 
-def test_objective_device_and_px_size(main_window: MainWindow):
-    mmc = main_window._mmc
+def test_objective_widget(global_mmcore: CMMCorePlus, qtbot):
+
+    obj_wdg = MMObjectivesWidget()
+    qtbot.addWidget(obj_wdg)
+
+    assert obj_wdg._combo.currentText() == "Nikon 10X S Fluor"
+    with pytest.raises(ValueError):
+        obj_wdg._combo.setCurrentText("10asdfdsX")
 
-    main_window.obj_wdg.objective_comboBox.setCurrentText("10X")
-    assert main_window.obj_wdg.objective_comboBox.currentText() == "10X"
-    assert mmc.getCurrentPixelSizeConfig() == "Res10x"
-    assert main_window.cam_wdg.px_size_spinbox.value() == 1.0
+    assert global_mmcore.getCurrentPixelSizeConfig() == "Res10x"
 
-    main_window.cam_wdg.px_size_spinbox.setValue(6.5)
```

Comment:
note, removal of this line is temporarily removing the custom pixelSizeConfig functionality